pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -300,6 +300,9 @@ const (
 	// ShootOperationRotateCAComplete is a constant for an annotation on a Shoot indicating that the rotation of the
 	// certificate authorities shall be completed.
 	ShootOperationRotateCAComplete = "rotate-ca-complete"
+	// ShootOperationRotateObservabilityUserCredentials is a constant for an annotation on a Shoot indicating that the credentials
+	// contained in the observability-user secret shall be rotated.
```

Comment:
```suggestion
	// ShootOperationRotateObservabilityCredentials is a constant for an annotation on a Shoot indicating that the credentials
	// for the observability stack secret shall be rotated. Note that this only affects the user credentials since the operator credentials are rotated automatically each `30d`.
```